### DISCUSSION TITLE
Bluetooth connects for one second and then disconnects

### DISCUSSION THREAD

**Discussion by nleric** [blueman-project/blueman#2422] [Q&A]
blueman:
BlueZ:
Distribution:
Desktop environment:

* [ ] I have consulted the [Troubleshooting page](https://github.com/blueman-project/blueman/wiki/Troubleshooting) and done my best effort to follow.

Hi, I am using Zorin 17 Pro and when I connect Bluetooth earbuds it connects for one second and then disconnects, any idea how to get Bluetooth working?
Thanks,
Noel

**Comment by issabinho57** [comment]
Am also experiencing the same issue here. I tried to even restart my laptop but it seems the problem still persists.

**Comment by vensoran-studios** [comment]
This happens to me too, and even after restarting the device, the problem still persists on Linux Mint. To work around this, I removed the device from history, searched again, connected it after finding it, and it worked.
